src/Factory/Api/Mastodon/Status.php

```diff
@@ -76,15 +77,16 @@ public function __construct(LoggerInterface $logger, Database $dba,
 	}
 
 	/**
-	 * @param int  $uriId  Uri-ID of the item
-	 * @param int  $uid    Item user
-	 * @param bool $reblog Check for reblogged post
+	 * @param int  $uriId           Uri-ID of the item
+	 * @param int  $uid             Item user
+	 * @param bool $reblog          Check for reblogged post
+	 * @param bool $in_reply_status Add an "in_reply_status" element
 	 *
 	 * @return \Friendica\Object\Api\Mastodon\Status
 	 * @throws HTTPException\InternalServerErrorException
 	 * @throws ImagickException|HTTPException\NotFoundException
 	 */
-	public function createFromUriId(int $uriId, int $uid = 0, bool $reblog = true): \Friendica\Object\Api\Mastodon\Status
+	public function createFromUriId(int $uriId, int $uid = 0, bool $reblog = true, bool $in_reply_status = true, bool $diplay_quote = false): \Friendica\Object\Api\Mastodon\Status
```

Comment:
Please also document the new `$display_quote` parameter.
```suggestion
	public function createFromUriId(int $uriId, int $uid = 0, bool $reblog = true, bool $in_reply_status = true, bool $display_quote = false): \Friendica\Object\Api\Mastodon\Status
```

Reply:
Done